__tests__/app.test.js

```diff
@@ -32,142 +32,263 @@ describe("GET/api/topics", () => {
+        .expect(404)
+        .then(({ body: comments }) => {
+          expect(comments.msg).toBe("Id not found");
+        });
+    });
+    test("400: return error wrong type", () => {
+      return request(app)
+        .get("/api/articles/nathanielwashere/comments")
+        .expect(400)
+        .then(({ body: comments }) => {
+          expect(comments.msg).toBe("Bad Request");
+        });
+    });
+    describe("PATCH/api/articles/:article_id", () => {
+      test("200: return article by id with increased vote value", () => {
         return request(app)
-          .get("/api/articles/1")
+          .patch("/api/articles/1")
+          .send({ inc_votes: 10 })
           .expect(200)
-          .then(({ body: article }) => {
+          .then(({ body: { article } }) => {
             expect(article).toEqual({
               article_id: 1,
               title: "Living in the shadow of a great man",
               topic: "mitch",
               author: "butter_bridge",
               body: "I find this existence challenging",
               created_at: "2020-07-09T20:11:00.000Z",
-              votes: 100,
-              comment_count: 11,
-            });
-          });
-      });
-      describe("PATCH/api/articles/:article_id", () => {
-        test("200: return article by id with increased vote value", () => {
-          return request(app)
-            .patch("/api/articles/1")
-            .send({ inc_votes: 10 })
-            .expect(200)
-            .then(({ body: { article } }) => {
-              expect(article).toEqual({
-                article_id: 1,
-                title: "Living in the shadow of a great man",
-                topic: "mitch",
-                author: "butter_bridge",
-                body: "I find this existence challenging",
-                created_at: "2020-07-09T20:11:00.000Z",
-                votes: 110,
-              });
+              votes: 110,
             });
-        });
-        test("400: inc_votes is not a number", () => {
-          return request(app)
-            .patch("/api/articles/1")
-            .send({ inc_votes: "banana" })
-            .expect(400)
-            .then(({ body }) => {
-              expect(body.msg).toBe("Bad Request")
-            })
-        })
-      });
-      test("400: article id not in database", () => {
-        return request(app)
-          .get("/api/articles/1123")
-          .expect(400)
-          .then(({ body }) => {
-            expect(body.msg).toBe("Id not found");
           });
       });
-      test("400: wrong type of data", () => {
+      test("400: inc_votes is not a number", () => {
         return request(app)
-          .get("/api/articles/banana")
+          .patch("/api/articles/1")
+          .send({ inc_votes: "banana" })
           .expect(400)
           .then(({ body }) => {
             expect(body.msg).toBe("Bad Request");
           });
       });
     });
+    test("400: article id not in database", () => {
```

Comment:
I dont really understand why all these changes are here, it feels like you're adding code that's not in the ticket somehow?